unstructured/documents/elements.py

```diff
@@ -151,12 +151,13 @@ class ElementMetadata:
 
     # Metadata extracted via regex
     regex_metadata: Optional[Dict[str, List[RegexMetadata]]] = None
+    include_path_in_metadata_filename: InitVar[Optional[bool]] = None
```

Comment:
I'd prefer not to have this a a value in `ElementMetadata`. I don't think we'd need this if we handle setting the filename and directory name in the `process_metadata` decorator.

Reply:
That would also eliminate the need for the `InitVar` logic you added to `weaviate.py`

Reply:
`InitVar` will not be presented in ElementMetadata (I mean `print(element.metadata)` - will not include with value) it is just for __post_init__ function

Reply:
> I'd prefer not to have this a a value in `ElementMetadata`. I don't think we'd need this if we handle setting the filename and directory name in the `process_metadata` decorator.

InitVar will not be presented in ElementMetadata (I mean print(element.metadata) - will not include with value) it is just for post_init function